src/components/State/Store.tsx

```diff
@@ -47,6 +47,15 @@ type Store = {
     showHiddenTags: boolean;
     setShowHiddenTags: (show: boolean) => void;
 
+    tags: { tagId: string, tagName: string, newValue: string }[];
+    setTags: (tags: { tagId: string, tagName: string, newValue: string }[]) => void;
```

Comment:
```suggestion
    setTags: (tags: <type name>) => void;
```
same as above - make a type dec in a types file and import it